doc/examples/effects/infinite_effects/lib/main.dart

```diff
@@ -18,9 +18,9 @@ void main() async {
 }
 
 class MyGame extends BaseGame with TapDetector {
-  Square greenSquare;
-  Square redSquare;
-  Square orangeSquare;
+  Square? greenSquare;
+  Square? redSquare;
+  Square? orangeSquare;
```

Comment:
should these all be late? or even move the initializer to here

Reply:
```dart
  MyGame()
      : greenSquare =
            Square(Paint()..color = const Color(0xAA338833), Vector2.all(100)),
        redSquare =
            Square(Paint()..color = const Color(0xAA883333), Vector2.all(200)),
        orangeSquare = Square(
            Paint()..color = const Color(0xAABB6633), Vector2(200, 400)) {
    add(greenSquare);
    add(redSquare);
    add(orangeSquare);
  }
```

I'd rather do `late` than that.